src/v/rpc/rpc_compiler.py

```diff
@@ -124,9 +124,7 @@ class failure_probes;
       });
     }
     virtual ss::future<{{method.output_type}}>
-    {{method.name}}({{method.input_type}}, ::rpc::streaming_context&) {
-       throw std::runtime_error("unimplemented method");
-    }
+    {{method.name}}({{method.input_type}}, ::rpc::streaming_context&) = 0;
```

Comment:
huh, i'm surprised its lasted this long. maybe the typical workflow for adding new rpc service/methods tends to make it unlikely that this is not implemented?